packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -263,6 +262,21 @@ export class BridgeHelper {
     )
   }
 
+  static getOutboundTransferData = async (
+    gatewayAddress: string,
+    provider: providers.Provider,
+    filter: ethers.providers.Filter = {}
+  ) => {
+    const contract = L1ERC20Gateway__factory.connect(gatewayAddress, provider)
+    const logs = await BridgeHelper.getEventLogs(
+      'OutboundTransferInitiated',
+      contract,
+      [],
+      filter
+    )
+    return logs
+  }
+
   public static getEventLogs = (
```

Comment:
I think we should use type safe alternatives even if we make our code a bit more WET.
typechain interfaces cover most of the boilerplate code

Reply:
agreed

Reply:
(separate PR)